sdmetrics/single_table/new_row_synthesis.py

```diff
@@ -70,16 +70,16 @@ def compute_breakdown(cls, real_data, synthetic_data, metadata=None,
                 synthetic_data = synthetic_data.sample(n=synthetic_sample_size)
 
         numerical_fields = []
-        discrete_fields = []
-        for field, field_meta in metadata['fields'].items():
-            if field_meta['type'] == 'datetime':
+
+        for field, field_meta in cls._get_columns_from_metadata(metadata).items():
+            field_type = cls._get_type_from_column_meta(field_meta)
+
+            if field_type == 'datetime':
                 real_data[field] = pd.to_numeric(real_data[field])
                 synthetic_data[field] = pd.to_numeric(synthetic_data[field])
                 numerical_fields.append(field)
-            elif field_meta['type'] == 'numerical':
+            elif field_type == 'numerical':
                 numerical_fields.append(field)
-            else:
```

Comment:
why did we get rid of this case?

Reply:
The `discrete_fields` variable wasn't being used. In the code later on, we just assume everything not in `numerical_fields` is discrete.